app/assets/javascripts/mumuki_laboratory/application/gamification.js

```diff
@@ -0,0 +1,65 @@
+class Formula {
```

Comment:
Great. Please notice all JS classes should be namespaced into `mumuki` or submodules.